Userland/Libraries/LibELF/DynamicLoader.cpp

```diff
@@ -803,8 +803,8 @@ void DynamicLoader::call_object_init_functions()
     typedef void (*InitFunc)();
 
     if (m_dynamic_object->has_init_section()) {
-        auto init_function = m_dynamic_object->init_section_function();
-        (init_function)();
+        dbgln("DynamicLoader: Object has obsolete init_section {}", filepath());
```

Comment:
nit: I would say "Object has obsolete DT_INIT dynamic array tag". init_section is just our name for the address pointed to by DT_INIT.

Reply:
Right, will update.